compiler/onecc-docker/README.md

```diff
@@ -0,0 +1,36 @@
+# onecc-docker
+
+_onecc-docker_ is designed to improve user usability.
+
+## Description
+
+For now, `one-cmds` tools only support Ubuntu 18.04 and 20.04(not officially).
+So, It is difficult for people in different environments to use our tools without using ubuntu 18.04.
+
+To overcome this limitation, we provide _onecc-docker_ that runs using a docker so that users can use `one-cmds` more widely.
+
+This tool aims at the following objectives.
+
+- Unsupported Ubuntu OS supports ONE tool
+- Different versions of ONE tools can be used comfortably
+- Install and use ONE tools lightly and quickly using Docker
```

Comment:
These two lines seems to say the same thing...maybe just use `Install and use ONE tools lightly and quickly using Docker`